application/F3DOptionsTools.cxx

```diff
@@ -188,20 +188,28 @@ std::string CollapseName(const std::string_view& longName, const std::string_vie
 }
 
 //----------------------------------------------------------------------------
-void PrintHelpPair(
-  std::string_view key, std::string_view help, int keyWidth = 10, int helpWidth = 70)
+void PrintHelpPair(std::string_view key, std::string_view help, int keyWidth = 50)
 {
   std::stringstream ss;
-  ss << "  " << std::left << std::setw(keyWidth) << key << " " << std::setw(helpWidth) << help;
+  if (key.find(' ') != std::string::npos)
+  {
+    ss << key << "\n";
+    ss << std::setw(keyWidth) << " " << help;
+  }
+  else
+  {
+    ss << std::left << std::setw(keyWidth) << key;
+    ss << " " << help;
+  }
```

Comment:
this is not what @snoyer suggested here: https://github.com/f3d-app/f3d/pull/1626#discussion_r1760686326

Lets implement what was suggested.